hello_world/src/hello-world.cpp

```diff
@@ -1,5 +1,5 @@
 #include "hello-world.hpp"
-#include "cmakepublic/cmake_public.hpp"
+#include "helloworld/external/cmakepublic/cmake_public.hpp"

```

Comment:
```suggestion
#include "cmakepublic/cmake_public.hpp"

```
We don't want to put the `helloworld/external` prefix on there because if the user provides their own copy of CMakePublic, it's not going to be in that directory. When we look for CMakePublic inside the hello world external project, the resulting `cmakepublic` target should have its include paths set up so that `#include "cmakepublic/cmake_public.hpp"`